src/mito2/src/manifest/manager.rs

```diff
@@ -333,15 +252,29 @@ impl RegionManifestManagerInner {
             last_version: version,
             last_checkpoint_version,
             manifest: Arc::new(manifest),
+            stopped: false,
         }))
     }
 
-    async fn stop(&mut self) -> Result<()> {
+    /// Stops the manager.
+    pub async fn stop(&mut self) -> Result<()> {
+        self.stopped = true;
```

Comment:
Maybe add some info log?

Reply:
There is already a log in `MitoRegion::stop()` which calls this method.